documentation/src/main/doc/antora.yml

```diff
@@ -0,0 +1,5 @@
+name: smallrye-reactive-messaging
+title: SmallRye Reactive Messaging
+version: ${project.version}
```

Comment:
Does this need to be a variable? I mean, even if your `project.version` is for example `1.3.5-SNAPSHOT`, the "documentation version" will probably be 1.3 (we will not produce a different doc version for each micro release or so)
What I was expecting to do with this is that we simply put in the `major.minor` version here and update it manually once in a while (which will basically be only when you create a new branch targeting a different minor version, so not very often)

Reply:
I caught it yesterday. I missed it.

The issue is that it avoids the release process automation without adding a commit in the process. Adding a commit is not impossible, just tricky as I will rely on regexp and replacement. 

Note that most of the time we are releasing minor, so it would require an increment every time (more or less).

Reply:
I found a way to extract this in Maven.